R/cv_varsel.R

```diff
@@ -797,10 +797,11 @@ loo_varsel <- function(refmodel, method, nterms_max, ndraws,
     }
     if (nrow(log_lik_ref) > 1) {
       # Take into account that clustered draws usually have different weights:
-      lw_sub <- log_lik_ref + log(refdist_eval$wdraws_prj)
+      lwref_lwdraws <- -log_lik_ref + log(refdist_eval$wdraws_prj)
```

Comment:
@avehtari, can you confirm that `-log_lik_ref` instead of `log_lik_ref` is correct?

Reply:
Yes. Earlier the negation was done when calling `importance_sampling_func()`, now you have moved the negation here.

Reply:
Great, thanks. Then I'll merge this.